### DISCUSSION TITLE
How to specify filename and path for "Export Data" block?

### DISCUSSION THREAD

**Discussion by mblais** [AutomaApp/automa#1248] [General]
If I try to specify a directory or path in the `File name` parameter of the `Export Data` block, it always causes an error (on Windows).

Is there any way to specify the export file's path in the block?

**RELATED:** 

If I execute the `Export Data` block multiple times in a loop, every time I see the Save As dialog, and/or a prompt to overwrite the existing file.

Is there any way to **force overwriting the exported data file without any prompting**, every time through the loop?

**Comment by ve3** [comment]
To let who is searching how to use **Export Data** know how to use it.

**Data to export**  
In case you save data as variable `$$html`.  
Enter variable name as `$$html`. Not `{{variables@$$html}}`.

**File name**  
Enter file name only without extension nor path. Example: `helloworld`.

On execute, it will be ask for location to save this file based on **Export as**, if **Export as** is text then it will be ask for location to save as **helloworld.txt**. 

I can't find the way to auto save or overwrite without any prompting either.
